posts/2021-10-01-ga-release-post.adoc

```diff
@@ -0,0 +1,175 @@
+---
+layout: post
+title: "Java 17 support and a number of significant bug fixes in Open Liberty 21.0.0.10"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/ryan-storey
+author_github: https://github.com/ryan-storey
+seo-title: Java 17 support and a number of significant bug fixes in Open Liberty 21.0.0.10 - OpenLiberty.io
+seo-description: The release of Open Liberty 21.0.0.10 offers the long-awaited Java 17 support and its many new enhancements and functionality!
+blog_description: The release of Open Liberty 21.0.0.10 offers the long-awaited Java 17 support and its many new enhancements and functionality!
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= Java 17 support and a number of significant bug fixes in Open Liberty 21.0.0.10
+Ryan Storey <https://github.com/ryan-storey>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+The release of Open Liberty 21.0.0.10 offers the long-awaited Java 17 support and its many new enhancements and functionality! This release also provides several important new bug fixes.
+
+In link:{url-about}[Open Liberty] 21.0.0.10:
+
+* <<java17, Java 17 support>>
+
+View the list of fixed bugs in link:https://github.com/OpenLiberty/open-liberty/issues?q=label%3Arelease%3A210010+label%3A%22release+bug%22
+
+[#run]
+== Run your apps using 21.0.0.10
+
+If you're using link:{url-prefix}/guides/maven-intro.html[Maven], here are the coordinates:
```

Comment:
Maven link does not work in staging

Reply:
This is expected behaviour don't worry, as the {url-prefix} is the hostname, i.e. https://openliberty.io, or in this case,  https://blogs-draft-openlibertyio.mybluemix.net. The draft site only works for blogs and therefore does not have the maven guide, whereas on the prod site the {url-prefix} will be https://openliberty.io and therefore will work.